readthedocs/projects/tasks/search.py

```diff
@@ -2,27 +2,29 @@
 
 import structlog
 
-from readthedocs.builds.constants import EXTERNAL
-from readthedocs.builds.models import Version
-from readthedocs.projects.models import HTMLFile, ImportedFile, Project
+from readthedocs.builds.constants import BUILD_STATE_FINISHED
+from readthedocs.builds.models import Build, Version
+from readthedocs.projects.models import HTMLFile, Project
 from readthedocs.projects.signals import files_changed
-from readthedocs.search.utils import index_new_files, remove_indexed_files
+from readthedocs.search.documents import PageDocument
+from readthedocs.search.utils import index_objects, remove_indexed_files
 from readthedocs.storage import build_media_storage
 from readthedocs.worker import app
 
 log = structlog.get_logger(__name__)
 
 
-@app.task(queue='reindex')
+# TODO: remove after deploy. This is kept, so builds keep
+# working while we deploy the task below.
```

Comment:
I've been trying to avoid using "this" on comments and making it more explicit mentioning the name of the thing we have to remove.

```suggestion
# TODO: remove `fileify` task after deploy. We keep it for now so builds keep
# working while we deploy the task below.
```

This is a nit review comment that don't even need to be merged, but wanted to mentioned it for the future.